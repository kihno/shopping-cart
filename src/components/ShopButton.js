import React from 'react';

const ShopButton = (props) => {
    const { handleShopClick } = props;

    return(
        <button type="button" id="shopButton" onClick={handleShopClick}></button>
    )
}

export default ShopButton;