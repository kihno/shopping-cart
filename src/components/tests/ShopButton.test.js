import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import ShopButton from '../ShopButton';

describe('Add to cart button', () => {
    it ('renders button', () => {
        const { container } = render(<ShopButton />);
        expect(container).toMatchSnapshot();
    });
});