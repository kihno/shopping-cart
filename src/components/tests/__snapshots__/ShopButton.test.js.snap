// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add to cart button renders button 1`] = `
<div>
  <button
    id="shopButton"
    type="button"
  />
</div>
`;
